.github/workflows/ci.yaml

```diff
@@ -103,3 +103,40 @@ jobs:
 
       - name: Check documentation
         run: cargo doc --locked --workspace --all-features --document-private-items --no-deps
+
+  bevy-lint:
+    name: Bevy linter
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      LINTER_TOOLCHAIN: nightly-2025-04-03
```

Comment:
This is specific to the linter version installed, right? If so, can we add a comment? Maybe a link to that fancy md table, noting that needs to be updated for future versions of Bevy?
Side note, maybe that could be encoded into an own GitHub workflow at some point :)

Reply:
Actually, let me just make this an action in the `bevy_cli` repository. That should work better!